lib/pointfree.mjs

```diff
@@ -0,0 +1,9 @@
+const pointfree = (m, arity = 1) => {
```

Comment:
Can you update `m` to be clearer (such as `methodName`)

Reply:
Sure. I thought I already did that. Whoops